### PR TITLE
improved issue template

### PR DESCRIPTION
**Pull request by samuelcolvin**
* improve bug template (use form)
* add feature request template - instead of directing to discussions
* add comment about my bug bear to PR template

Demo of issue templates is available [here](https://github.com/samuelcolvin/pydantic-sandbox/issues/new/choose)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        ...
+      render: Python
+
+  - type: textarea
+    id: version
+    attributes:
+      label: Python, Pydantic & OS Version
+      description: |
+        Which version Python & Pydantic are you using, and which Operating System?
```
Comment (hramezani) [inline .github/ISSUE_TEMPLATE/bug.yml:59]:
```suggestion
        Which version of Python & Pydantic are you using, and which Operating System?
```

**Review by hramezani — APPROVED** [review]
Other than small suggestion LGTM :+1:

**Comment by samuelcolvin** [comment]
@hramezani I've added some more questions, unless you have immediate feedback, let's start with this and see how it goes.

**Comment by hramezani** [comment]
> @hramezani I've added some more questions, unless you have immediate feedback, let's start with this and see how it goes.

@samuelcolvin I've checked them as well. LGTM 👍

**Pull request merged**